RELEASES.md

```diff
@@ -43,9 +43,54 @@ Specifications have a variety of different timelines in their lifecycle.
-  This means a major release like a v1.0.0 or v2.0.0 release will take 1 month at minimum: one week for rc1, one week for rc2, one week for rc3, and one week for the major release itself.
-  Maintainers SHOULD strive to make zero breaking changes during this cycle of release candidates and SHOULD restart the three-candidate count when a breaking change is introduced.
-  For example if a breaking change is introduced in v1.0.0-rc2 then the series would end with v1.0.0-rc4 and v1.0.0.
-- Minor and patch releases SHOULD be made on an as-needed basis.
+    This means a major release like a v1.0.0 or v2.0.0 release will take 1 month at minimum: one week for rc1, one week for rc2, one week for rc3, and one week for the major release itself.
+    Maintainers SHOULD strive to make zero breaking changes during this cycle of release candidates and SHOULD restart the three-candidate count when a breaking change is introduced.
+    For example if a breaking change is introduced in v1.0.0-rc2 then the series would end with v1.0.0-rc4 and v1.0.0.
+* Minor and patch releases SHOULD be made on an as-needed basis.
 
 [charter]: https://github.com/opencontainers/tob/blob/main/CHARTER.md
+
+## Checklist
+
+Releases usually follow a few steps:
+
+* [ ] prepare a pull-request for the release
+  * [ ] a commit updating `./ChangeLog`
+    * [ ] `git log --oneline --no-merges --decorate --name-status v1.0.1..HEAD | vim -`
+    * [ ] `:% s/(pr\/\(\d*\))\(.*\)/\2 (#\1)/` to move the PR to the end of line and match previous formatting
+    * [ ] review `(^M|^A|^D)` for impact of the commit
+    * [ ] group commits to `Additions:`, `Minor fixes and documentation:`, `Breaking changes:`
+    * [ ] delete the `(^M|^A|^D)` lines, `:%!grep -vE '(^M|^A|^D)'`
+    * [ ] merge multi-commit PRs (so each line has a `(#num)` suffix)
+    * [ ] drop hash and indent, `:'<,'> s/^\w*  /^I* /`
+  * [ ] a commit bumping `./specs-go/version.go` to next version and empty the `VersionDev` variable
+  * [ ] a commit adding back the "-dev" to `VersionDev`
+* [ ] send email to dev@opencontainers.org
+  * [ ] copy the exact commit hash for bumping the version from the pull-request (since master always stays as "-dev")
+  * [ ] count the PRs since last release (that this version is tracking, in the cases of multiple branching), like `git log --pretty=oneline --no-merges --decorate $priorTag..$versionBumpCommit  | grep \(pr\/ | wc -l`
+  * [ ] get the date for a week from now, like `TZ=UTC date --date='next week'`
+  * [ ] OPTIONAL find a cute animal gif to attach to the email, and subsequently the release description
+  * [ ] subject line like `[runtime-spec VOTE] tag $versionBumpCommit as $version (closes $dateWeekFromNowUTC)`
+  * [ ] email body like
+```
+Hey everyone,
+
+There have been $numPRs PRs merged since $priorTag release (https://github.com/opencontainers/runtime-spec/compare/$priorTag...$versionBumpCommit).
```

Comment:
```suggestion
There have been $numPRs PRs merged since $priorTag release (https://github.com/opencontainers/image-spec/compare/$priorTag...$versionBumpCommit).
```